torchtext/csrc/vocab.h

```diff
@@ -41,10 +41,9 @@ VocabStates _set_vocab_states(const c10::intrusive_ptr<Vocab> &self);
 Vocab _load_vocab_from_file(const std::string &file_path,
```

Comment:
`_load_vocab_from_file` and `_load_vocab_from_raw_text_file` are different. `_load_vocab_from_raw_text_file` requires a tokenizer, which works on the raw text file. `_load_vocab_from_file` will load the vocab from a column of vocabulary tokens saved in the text file.

Reply:
Yes and this naming is meant to disambiguate that by calling "_load_vocab_from_raw_text_file" "build_vocab_from_text_file" (using build just as we do for build_from_iterator) and by calling "_load_vocab_from_file" "load_vocab_from_text_file" similar to how we'd call torch.load to load a model (except here it's a specific plain-text format).